### ISSUE TITLE
App opening up blank

### ISSUE THREAD

**Issue by roynel** [tutao/tutanota#7288]
- [ ] After the new update for Linux my app is opening up blank.
- [ ] I've searched and did not find a similar issue.

**Describe the bug**
I've updated to the latest version using the latest .AppImage from the website and now when I open the app it just comes up blank. See screenshot.

**Expected behavior**
Expected it to open and see my emails.

**Screenshots**
![2024-07-26_11-08](https://github.com/user-attachments/assets/4df779cd-72f8-4e66-8708-f6b5543bb164)

**Desktop (please complete the following information):**
 - OS: Debian GNU/Linux 11 (bullseye)
 - Desktop Environment: Plasma 5.20.5
 - Version: bullseye

**Comment by rezbyte** [comment]
Hi @roynel ,
I am sorry to hear you are having issues with the client on Debian 11.

This might be related to #5653. Could you try deleting `~/.config/tutanota-desktop/GPUCache` on your machine and see if that remedies the issue?

Kind regards,
rezbyte

**Comment by roynel** [comment]
@rezbyte thanks rezbyte, I forgot to mention I actually saw this and did try deleting the GPUCache before creating this bug post however no luck. It opened the same as before just showing a blank screen.

**Comment by roynel** [comment]
@charlag here's the output from the terminal (F12 doesn't seem to be working)


```
the monkey has been patched pid: 15844
(node:15844) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
(Use `tutanota-desktop --trace-deprecation ...` to show where the warning was created)
retrying to create credentials db
deleting credentials db
version:   235.240718.0
MESA-INTEL: warning: Ivy Bridge Vulkan support is incomplete
(node:15844) UnhandledPromiseRejectionWarning: Error: /lib/x86_64-linux-gnu/libstdc++.so.6: version `GLIBCXX_3.4.29' not found (required by /tmp/.mount_tutanoFCpZtf/resources/app.asar.unpacked/desktop/better-sqlite3.node)
    at process.func [as dlopen] (node:electron/js2c/node_init:2:2559)
    at Module._extensions..node (node:internal/modules/cjs/loader:1470:18)
    at Object.func [as .node] (node:electron/js2c/node_init:2:2786)
    at Module.load (node:internal/modules/cjs/loader:1215:32)
    at Module._load (node:internal/modules/cjs/loader:1031:12)
    at c._load (node:electron/js2c/node_init:2:17025)
    at Module.require (node:internal/modules/cjs/loader:1240:19)
    at require (node:internal/modules/helpers:179:18)
    at new Fe (/tmp/.mount_tutanoFCpZtf/resources/app.asar/desktop/better-sqlite3-Bu0ZH0B8.js:1:18578)
    at _P.openDb (/tmp/.mount_tutanoFCpZtf/resources/app.asar/desktop/DesktopMain.js:23:124316)
(node:15844) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
config up to date
libva error: /usr/lib/x86_64-linux-gnu/dri/iHD_drv_video.so init failed
[DesktopMain] getting dictionaries from: https://app.tuta.com/desktop/dictionaries/
[2024-07-29T20:51:49.130Z] Create tray:true
[DesktopMain] spellcheck-dictionary-initialized en-US
```

**Comment by rezbyte** [comment]
> MESA-INTEL: warning: Ivy Bridge Vulkan support is incomplete

[Your GPU does not have full Vulkan support](https://gitlab.freedesktop.org/mesa/mesa/-/issues/8249). It could be worth trying to run the client with a software renderer instead. Does the same issue occur if you run Chrome? Try running the client with the `--disable-gpu` flag to confirm a driver problem.

> (node:15844) UnhandledPromiseRejectionWarning: Error: /lib/x86_64-linux-gnu/libstdc++.so.6: version `GLIBCXX_3.4.29' not found (required by /tmp/.mount_tutanoFCpZtf/resources/app.asar.unpacked/desktop/better-sqlite3.node)
>     at process.func [as dlopen] (node:electron/js2c/node_init:2:2559)
>     at Module._extensions..node (node:internal/modules/cjs/loader:1470:18)
>     at Object.func [as .node] (node:electron/js2c/node_init:2:2786)
>     at Module.load (node:internal/modules/cjs/loader:1215:32)
>     at Module._load (node:internal/modules/cjs/loader:1031:12)
>     at c._load (node:electron/js2c/node_init:2:17025)
>     at Module.require (node:internal/modules/cjs/loader:1240:19)
>     at require (node:internal/modules/helpers:179:18)
>     at new Fe (/tmp/.mount_tutanoFCpZtf/resources/app.asar/desktop/better-sqlite3-Bu0ZH0B8.js:1:18578)
>     at _P.openDb (/tmp/.mount_tutanoFCpZtf/resources/app.asar/desktop/DesktopMain.js:23:124316)
> (node:15844) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)

This is likely an unrelated issue (#7133) that should not cause rendering problems

> libva error: /usr/lib/x86_64-linux-gnu/dri/iHD_drv_video.so init failed

This appears to be the cause of the rendering problem. There seems to be an issue with the driver for your integrated GPU.  [This suggestion may help](https://github.com/intel/media-driver/issues/1298#issuecomment-1090086480). Otherwise upgrading or downgrading your driver might help too.

**Comment by roynel** [comment]
@rezbyte I tried running Tutanota with `--disable-gpu` and still having the same issue. Here is the output from the terminal: 

```
the monkey has been patched pid: 6382
(node:6382) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
(Use `tutanota-desktop --trace-deprecation ...` to show where the warning was created)
retrying to create credentials db
deleting credentials db
version:   235.240718.0
MESA-INTEL: warning: Ivy Bridge Vulkan support is incomplete
(node:6382) UnhandledPromiseRejectionWarning: Error: /lib/x86_64-linux-gnu/libstdc++.so.6: version `GLIBCXX_3.4.29' not found (required by /tmp/.mount_tutanoj2QF5J/resources/app.asar.unpacked/desktop/better-sqlite3.node)
    at process.func [as dlopen] (node:electron/js2c/node_init:2:2559)
    at Module._extensions..node (node:internal/modules/cjs/loader:1470:18)
    at Object.func [as .node] (node:electron/js2c/node_init:2:2786)
    at Module.load (node:internal/modules/cjs/loader:1215:32)
    at Module._load (node:internal/modules/cjs/loader:1031:12)
    at c._load (node:electron/js2c/node_init:2:17025)
    at Module.require (node:internal/modules/cjs/loader:1240:19)
    at require (node:internal/modules/helpers:179:18)
    at new Fe (/tmp/.mount_tutanoj2QF5J/resources/app.asar/desktop/better-sqlite3-Bu0ZH0B8.js:1:18578)
    at _P.openDb (/tmp/.mount_tutanoj2QF5J/resources/app.asar/desktop/DesktopMain.js:23:124316)
(node:6382) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
resolving key... tutanota-vault
config up to date
[2024-08-13T13:44:31.383Z] using safeStorage with backend kwallet5
[DesktopMain] getting dictionaries from: https://app.tuta.com/desktop/dictionaries/
[2024-08-13T13:44:31.510Z] Create tray:true
[SSEFacade] last missed notification check: { lastMissedNotificationCheckTime: null }
[ApplicationWindow] webAssetsPath:  /tmp/.mount_tutanoj2QF5J/resources/app.asar
libva error: /usr/lib/x86_64-linux-gnu/dri/iHD_drv_video.so init failed
default mailto handler: false
Webapp ready
[SSEFacade] No SSE info
[2024-08-13T13:44:31.679Z] [ElectronUpdater] INFO: Checking for update
[2024-08-13T13:44:31.679Z] [ElectronUpdater] INFO: checking-for-update
[DesktopMain] spellcheck-dictionary-initialized en-US
/home/roynel/.local/share/applications/tutanota-desktop.desktop does not exist, checking for permission to ask for permission...
[DesktopMain] getting dictionaries from: https://app.tuta.com/desktop/dictionaries/
[DesktopMain] spellcheck-dictionary-initialized en-US
[2024-08-13T13:44:32.617Z] [ElectronUpdater] INFO: Update for version 235.240718.0 is not available (latest version: 235.240718.0, downgrade is disallowed).
[2024-08-13T13:44:32.617Z] [ElectronUpdater] INFO: update not available: {
  version: '235.240718.0',
  files: [
    {
      url: 'tutanota-desktop-linux.AppImage',
      sha512: 'qrP0BCDJJMr9NJSCC9zdmAEomW4TXLooftxJOpPCbmnI/N6pehsFYbaqh7OQOGq4d9/U5fj5pyO4MZS7rOsUwg==',
      size: 127302353,
      blockMapSize: 133861
    }
  ],
  path: 'tutanota-desktop-linux.AppImage',
  sha512: 'qrP0BCDJJMr9NJSCC9zdmAEomW4TXLooftxJOpPCbmnI/N6pehsFYbaqh7OQOGq4d9/U5fj5pyO4MZS7rOsUwg==',
  releaseDate: '2024-07-19T11:46:10.394Z',
  signature: 'McW8KnxVUVuua2UMt4ZPYyLw0l0c7tmBZknHBIyqbs/EmBeQE8H0mY1+DKfdiKiolUYQL6M+w6TzNkawmiuIxRJ/gvJWLj52R2RmhRUrxzqkqBQL1VK1DJV6RfrbvD/1SKcaUih6cnKuDligPLgtUmj79XbP1uX25B4DpfnHl2hwBdN7uD8wbpv7Mwys+GlmOhJ10hUgurHjw6a5Ci9j0Mg1ejO9XvwCPQ2EcKMIoQ2q4JmzMqBfUjKb4LDBpKy2MxWS6s1qgKVG+jNh5gug5UDLBYljS0iX9PmgJYOJgu4vcqnfiF0uPoSm0qQKbo0cnBhm2gnuVIfrEgaFbUiOMQ=='
}
```

**Comment by roynel** [comment]
@rezbyte @charlag still no response and it is still not working, do you guys have any resolution for this yet? I tried the disable gpu tag and still no luck.

**Comment by charlag** [comment]
@roynel it seems like the issue is the GLIBC version. It was fixed in v240.240813.0. Please re-download the client from the website.

**Comment by roynel** [comment]
Thank you @charlag! Fully working now!
